jobs/apptypes_test.groovy

```diff
@@ -0,0 +1,56 @@
+evaluate(new File("${WORKSPACE}/common.groovy"))
+
+import utilities.StatusUpdater
+
+
+name = "apptypes_e2e"
+
+
+  job(name) {
+    description """
+      Watches the e2e-runner repo for a ${config.type} commit
```

Comment:
`config.type` won't be defined... should just change the description here to what this new job will do